### ISSUE TITLE
HTML should be branched out too

### ISSUE THREAD

**Issue by binary-solo** [kamranahmedse/developer-roadmap#387]
I think you might consider expanding a little on some aspects of HTML. Mainly -- importance of semantics, accessibility, and progressive enhancement should be stressed out.

**Comment by Ayush-Rawal** [comment]
I second that, things like ARIA, using tags for page-structure instead of functionality etc.. should definitely have a place

**Comment by kamranahmedse** [comment]
Frontend roadmap has been rewritten from the ground up. You may have a look
